source/includes/v1/records_api.md

```diff
@@ -253,7 +253,7 @@ Parameter            | Description
 `exclude.<FACET>`    | Exclude records where `FACET` has the specified value from the result set. It can be used several times for the same facet or for different facets
 `fields`             | Restricts field to retrieve. This parameter accepts multiple field names separated by commas. Example: `fields=field1,field2,field3`
 `pretty_print`       | If set to true (default is false), pretty prints JSON and JSONP output
-`format`             | Format of the response output. Can be `json` (default), `jsonp`, `csv`, `geojson`, `geojsonp`
+`format`             | Format of the response output. Can be `csv` (default), `json`, `jsonp`, `geojson`, `geojsonp`
```

Comment:
This is for the geocluster endpoint isn't it? I thought the fix was for the download endpoint.
On the same note, the geocluster endpoint doesn't take a `format` parameter at all as far as I know...

Reply:
Good catch, wrong place, I'm tired... And you're right, geocluster doesn't take any format. I remove these and fixes the the download format